oss-s3/src/test/java/dev/ocpd/oss/AwsS3FileStoreTest.kt

```diff
@@ -0,0 +1,27 @@
+package dev.ocpd.oss
+
+import cloud.localstack.Localstack
+import cloud.localstack.docker.annotation.LocalstackDockerConfiguration
+import dev.ocpd.oss.config.AwsPropertiesEnvInitializer
+import dev.ocpd.oss.config.AwsS3FileStoreAutoConfiguration
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeAll
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.ContextConfiguration
+
+@SpringBootTest(classes = [AwsS3FileStoreAutoConfiguration::class])
+@ContextConfiguration(initializers = [AwsPropertiesEnvInitializer::class])
+object AwsS3FileStoreTest : FileStoreTest() {
+
+    @JvmStatic
```

Comment:
use `@TestInstance(PER_CLASS)`

Reply:
Localstack docker beans do not work with `PER_CLASS` JUnit Lifecycle.

see
https://github.com/localstack/localstack-java-utils/issues/104